src/app/index.ts

```diff
@@ -29,6 +29,7 @@ let language;
 const manifest = 'manifest';
 const sso = 'single-sign-on';
 const typescript = `TypeScript`;
+let jsonData;
```

Comment:
Why pull this out of the try block?

Reply:
It is being used on another function. So I wanted to let this other function `configureProject` also have access to `jsonData`